os/arch/arm/src/amebasmart/amebasmart_idle.c

```diff
@@ -40,6 +40,18 @@
 #endif
 
 #ifdef CONFIG_PM
+
+struct platform_pg_ops system_pg_ops = {
```

Comment:
minor, how about like?
```suggestion
struct pm_sleep_ops rtl8730e_sleep_ops = {
```